### ISSUE TITLE
Option to move session name

### ISSUE THREAD

**Issue by omerchn** [niksingh710/minimal-tmux-status#7]
Hello! I really love your theme, there is only one thing that would make it perfect for me:
would it be possible to add an option to move the session name to the left of the window names?
Thank you!

![image](https://github.com/niksingh710/minimal-tmux-status/assets/73636904/098cfe6e-f9d0-43c6-bf8d-e7a55cc9d68b)

**Comment by niksingh710** [comment]
Now this should be solved after the commit [8677c29](https://github.com/niksingh710/minimal-tmux-status/commit/8677c29003f95695970eb6f4b7793158c9baf1a0)

update to the latest push `prefix+U`
and then add this in the tmux.conf. 

```
set -g @minimal-tmux-right false
set -g @minimal-tmux-status-left-extra " #S "

```

Feel free to reopen if any issue.

These two variables should get you the result you want.
You may also need `set -g status-left-length 20` 

> Hello! I really love your theme

Thanks for the compliment

**Comment by omerchn** [comment]
Working perfectly! Thank you! 🙏
